packages/mdc-datatable/_variables.scss

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "@material/theme/functions";
+
+$mdc-datatable-fill-color: surface !default;
+$mdc-datatable-header-row-fill-color: inherit !default;
+$mdc-datatable-row-fill-color: inherit !default;
+$mdc-datatable-selected-row-fill-color: rgba(mdc-theme-prop-value(primary), .04) !default;
+
+$mdc-datatable-checked-icon-color: primary !default;
+$mdc-datatable-divider-color: rgba(mdc-theme-prop-value(on-surface), .12) !default;
+$mdc-datatable-divider-size: 1px !default;
+$mdc-datatable-row-hover-fill-color: rgba(mdc-theme-prop-value(on-surface), .04) !default;
+
+$mdc-datatable-header-row-text-color: rgba(mdc-theme-prop-value(on-surface), .87) !default;
+$mdc-datatable-row-text-color: rgba(mdc-theme-prop-value(on-surface), .87) !default;
+
+$mdc-datatable-shape-radius: medium !default;
+$mdc-datatable-stroke-size: 1px !default;
+$mdc-datatable-stroke-color: rgba(mdc-theme-prop-value(on-surface), .12) !default;
+
+$mdc-datatable-header-row-height: 48px !default;
+$mdc-datatable-row-height: 48px !default;
+$mdc-datatable-cell-leading-padding: 16px !default;
+$mdc-datatable-cell-trailing-padding: 16px !default;
```

Comment:
Is this supposed to be 12? https://material.io/design/components/data-tables.html#specs

Reply:
The spec seems bit outdated. These baseline values are blessed by designer.